src/hardware/developing-an-iox/protocol.md

```diff
@@ -281,6 +281,18 @@ Sent from the IOX to the GO device to inform the GO device of events or status c
 | 0-1 | 0x0004 |
 | 2 | Unused |
 
+#### Information Type 11 IOX Third-Party Tx-Buffer State (GO9+ only)
+The IOX sends a "Stop-sending" message to GO for every Tx Data(0x0B) message it receives when the IOX Tx-queue going to an external third-party device is about to fill-up.
+When the GO receives the  "Stop-sending" message, it should pause sending messages out from the GO Tx-message queue to the IOX. 
+When the IOX Tx-queue going to an external third-party device returns to the level that it can accept more messages, the IOX will send out a "Resume-sending" message to the GO. 
+When the GO receives the "Resume-sending" message, it will resume sending messages out from the GO Tx-message-queue to the IOX.
+For downloading binary passthrough data to an external third-party device, in the case when the server is sending data faster than the GO can send out to the IOX, the GO Tx-message queue to the IOX will eventually fill-up and data may be dropped.
+
+| Parameter Type | Description |
+| --- | --- |
+| 0-1 | 0x000B |
+| 2 | Status:<br>0 = "Resume-sending": The IOX can accept more messages, the GO can resume sending passthrough messages to the Third-Party device.<br>1 = "Stop-sending": The IOX Tx-queue going to an external third-party device is about to fill-up. |
```

Comment:
Remove references to "external third-party device". That's what we use it for, but not necessarily what someone creating a custom IOX would use it for.

Reply:
Reword the descriptions.